postgres-json-schema--0.1.1.sql

```diff
@@ -237,6 +237,28 @@ BEGIN
     END IF;
   END IF;
 
+  IF schema ? 'format' AND jsonb_typeof(data) = 'string' THEN
+    DECLARE
+      target text := (data #>> '{}');
+      EMAIL_RX constant text := '^[\w.!#$%&''*+/=?^`{|}~-]+@[A-Z0-9](?:[A-Z0-9-]{0,61}[A-Z0-9])?(?:\.[A-Z0-9](?:[A-Z0-9-]{0,61}[A-Z0-9])?)*$';
```

Comment:
Case sensitive check is faster, so why not use the full case sensitive regex here and `target !~ EMAIL_RX` below?

```suggestion
      EMAIL_RX constant text := '^[a-zA-Z0-9.!#$%&''*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$';
```

Reply:
Fine. Longer and maybe more difficult to read but 10% are always 10%.